manifests/plugin/abrt.pp

```diff
@@ -30,7 +33,8 @@
 #
 class foreman_proxy::plugin::abrt (
   $enabled                 = $::foreman_proxy::plugin::abrt::params::enabled,
-  $group                   = $::foreman_proxy::plugin::abrt::params::group,
+  $version                 = $::foreman_proxy::plugin::version,
+  $group                   = $::foreman_proxy::user,
```

Comment:
This makes me cry now :). It seems a recent push in the master branch removed those:

    $group_real = pick($group, $::foreman_proxy::user)
    validate_string($group_real)

Should I add them again?